src/taipy/templates/scenario-management/{{cookiecutter.__root_folder_name}}/{{cookiecutter.__main_file}}.py

```diff
@@ -11,18 +11,20 @@ def on_init(state):
 
 
 def on_change(state, var, val):
-    state["scenario"].on_change(state, var, val)
-
     if var == "selected_scenario" and val:
         state.selected_scenario = val  # BUG
-        state["scenario"].manage_data_node_partials(state)
+        state.selected_data_node = None
+    if var == "selected_data_node" and val:
+        state.selected_data_node = val  # BUG
```

Comment:
Sorry I've just notice this now. What is the BUG here?

Reply:
I can't find the related issue; I have talked with Fred about this. Maybe, I still need to create it. The issue is that when the scenario or data node are changed, their value are None (`state.selected_data_node` and `state.selected_scenario`) instead of their true value

Reply:
Can you create a ticket, and add a TODO item to update the template later when the bug is fixed?